js/tic_tac_toe.js

```diff
@@ -1,2 +1,169 @@
 // Your code here
 // Use the jQuery
+
+$(document).ready(function(){
+  var theGame = {};
```

Comment:
I'm a little unclear on the purpose of this `theGame` object. I'd base things around a nested array.